zbench.zig

```diff
@@ -147,9 +147,11 @@ pub const Benchmark = struct {
         benchmark: anytype,
         config: Optional(Config),
     ) !void {
+        const O = @TypeOf(benchmark);
```

Comment:
Why `O`? And given it's only used in one place, I'm not sure that binding it to a name at all is an improvement.

Reply:
That was unintentional commit when I was trying to debug why .Pointer was not compiling.